templates/includes/sidebar.html

```diff
@@ -1,4 +1,20 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      <a href="{% url 'taxi:driver-detail' user.id %}">Driver: {{ user.first_name }}</a>
+    </li>
+    <li>
+      <a href="{% url 'logout' %}">Log out</a>
+    </li>
+    <br>
+    
+    {% else %}
+    <li>
+      <a href="{% url 'login' %}">Login</a>
+    </li>
+    <br>
```

Comment:
Similar to the previous comment, consider using CSS for spacing instead of `<br>` tags.